### DISCUSSION TITLE
Collaborators not able to clone repo

### DISCUSSION THREAD

**Discussion by athiavinash32** [community/community#143664] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have created a private repo, added collaborators, they not able to clone repo, one is getting as he doesn't have write access and the other is getting as repo is not found.
Please help me.

**Comment by JoaoCoelho2003** [comment]
Hello! It sounds like there may be a permissions issue with your private repository.

Here are a few things you can check out:

-  Make sure that you’ve given the correct permissions to each collaborator. If they need to clone the repository, ensure they have at least read access (or write access if they need to push changes).
-  If a collaborator is seeing “repository not found,” it’s possible they haven’t accepted the invitation to join the repo yet. They should check their email or GitHub notifications to make sure they’ve accepted your invitation.
- If nothing else works you can always remove all collaborators and re-invite them to the repository.

Hope this helps!

**Comment by ankurmalkani-gauri** [comment]
did you resolve? How/?

**Comment by PushoDev** [comment]
---

### Collaborator Who Cannot Find the Repository

The error message indicating that the repository cannot be found may be due to several reasons:

#### Possible Causes and Solutions:

1. **The collaborator has not accepted the invitation:**
   - When you add a collaborator, GitHub sends an invitation via email. The collaborator must accept the invitation before they can access the repository.
   - Ask the collaborator to check their inbox (and also the spam folder) to find the invitation and accept it.

2. **The repository is private, and the collaborator does not have access:**
   - If the collaborator has not been added correctly as a collaborator, they will not be able to see or clone the repository.
   - Verify again that the collaborator is in the list of **Collaborators and teams** in the repository settings.

3. **Error in the repository URL:**
   - Ensure that the collaborator is using the correct URL to clone the repository. For a private repository, the URL should be something like:
     ```
     https://github.com/your-username/your-repository.git
     ```
     or, if using SSH:
     ```
     git@github.com:your-username/your-repository.git
     ```
   - If using SSH, ensure that the collaborator has correctly configured their SSH key in their GitHub account. They can follow [this official guide](https://docs.github.com/en/authentication/connecting-to-github-with-ssh) to set it up.

---

#### Additional Steps:
- If the issue persists after following these steps, ask the collaborators to share the exact command they are using to clone the repository and the full error message. This will help diagnose the problem better.
- You can also refer to the official GitHub documentation on [managing collaborators](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories) for more details.

I hope this helps resolve the issues. If you have more questions, feel free to ask.

Good luck with your project!

--- 

Or send link collaborator, accepted the invitation
